docs/cita/system/store.md

```diff
@@ -0,0 +1,153 @@
+---
+id: store
+title: 存证
+---
+
+## 简述
+
+存证功能是将 **数据信息** 通过交易的方式存储到区块链上。
+
+### 合约信息
+
+合约地址为: `0xffffffffffffffffffffffffffffffffff010000`
+
+注：
+该合约地址没有合约方法，CITA 系统会将所有调用该地址的交易默认为 **存证** 交易。
+
+## 操作示例
+
+*首先需要启动一条链*
+
+通过以下命令初始化一条链，生成配置文件：
+
+```bash
+$ bin/cita create \
+    --super_admin "0x4b5ae4567ad5d9fb92bc9afd6a657e6fa13a2523" \
+    --nodes "127.0.0.1:4000,127.0.0.1:4001,127.0.0.1:4002,127.0.0.1:4003"
+```
+
+启动链接下来的步骤见[快速入门](../getting-started)部分。
+
+为了简化操作，以下将用 [cita-cli](https://github.com/cryptape/cita-cli) 命令行模式进行演示。
+所涉及的 JSON-RPC 接口，将在过程中做简要的说明，详细 JSON-RPC 说明，请参考 [这里](../rpc-guide/rpc.md)。
+
+### 发送数据存证交易
+
+*使用默认私钥进行演示*
+
+* 发送交易
+
+```bash
+cita> store data --content 0x3474524832041587483261437874876510837787487658976141347658716817365465984753489650 --private-key 0x5f0258a4778057a8a7d97809bd209055b2fbafa654ce7d31ec7191066b9225e6
```

Comment:
Maybe it's better to remove the `cita>` to unify style.(or add it on all other part about cli latter)

Reply:
ok, I will remove `cita>`